content/pt/docs/referencia-da-api/pix/relato-infracao/fechar-relato-infracao/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Fechar Relato de Infração"
+linkTitle: "Fechar Relato de Infração"
+date: 2022-03-08T15:17:00-03:00
+lastmod: 2022-03-08T15:20:00-03:00
+weight: 9
+description: >
+  
+---
+
+Através desse endpoint será possível que um participante indireto possa fechar um relato de infração.
```

Comment:
```suggestion
Através do seguinte endpoint, será possível que um participante indireto feche um relato de infração.
```

Reply:
Also I think it is important to mention all the requirements (`status` and `direction`) that an infraction report must fulfill in order to be able to close.